### ISSUE TITLE
Cannot authenticate account.

### ISSUE THREAD

**Issue by mjamison44** [homebridge-simplisafe3/homebridge-simplisafe3#320]
### Describe The Bug:
Authentication window opens as a blank white page and then quickly disappears. I was able to login once but have received the same error ever since. Now it won't let me unlink and attempt to relink my account. 

### To Reproduce:
Click on "Link Account."

### Expected behavior:
Enter Simplisafe credentials and have the plugin successfully login.

### Environment:

* **Plugin version**: 1.9.10
* **Node.js Version**: 16.14.0
* **NPM Version**: 1.9.10
* **Homebridge Version**: 1.4.0
* **Operating System**: Raspian

### Homebridge Config:
{
    "platform": "homebridge-simplisafe3.SimpliSafe 3",
    "sensorRefresh": 15,
    "refreshToken": ""
}

### Logs:
3/9/2022, 11:47:42 PMSimplisafe3 BridgeConfiguration change
3/9/2022, 11:47:42 PMsimplisafe3bridge stopping
3/9/2022, 11:47:47 PMSimplisafe3 Bridge starting
3/9/2022, 11:47:50 PMSimplisafe3 BridgeLoaded plugin 'homebridge-simplisafe3'
3/9/2022, 11:47:50 PMSimplisafe3 BridgeLoading 1 platforms...
3/9/2022, 11:47:51 PMSimplisafe3 Bridgehomebridge-simplisafe3.SimpliSafe 3ERRORSimpliSafe login failed with error: Error: No authentication credentials detected.
    at SimpliSafe3AuthenticationManager.refreshCredentials (/var/lib/hoobs/simplisafe3bridge/node_modules/homebridge-simplisafe3/lib/authManager.js:225:15)
    at new SS3Platform (/var/lib/hoobs/simplisafe3bridge/node_modules/homebridge-simplisafe3/index.js:80:41)
    at /usr/src/bridge/server.ts:455:46
    at Array.forEach (<anonymous>)
    at Server.loadPlatforms (/usr/src/bridge/server.ts:438:31)
    at Server.start (/usr/src/bridge/server.ts:239:69)
3/9/2022, 11:47:51 PMSimplisafe3 Bridgehomebridge-simplisafe3.SimpliSafe 3ERRORSee the plugin README for more information on authenticating with SimpliSafe.
3/9/2022, 11:47:51 PMSimplisafe3 Bridgehomebridge-simplisafe3.SimpliSafe 3ERRORInitial accessories refresh failed with error: Error: Not authenticated with SimpliSafe.
    at /var/lib/hoobs/simplisafe3bridge/node_modules/homebridge-simplisafe3/index.js:99:56
3/9/2022, 11:47:51 PMSimplisafe3 BridgeBridge is running on port 51826.

**Comment by mjamison44** [comment]
You think this is a Hoobs issue? How so? The Link Account button in the plugin isn’t logging me in correctly and won’t allow me to enter my credentials again.

**Comment by shamoon** [comment]
Because it works in homebridge AFAIK. HOOBS uses some workaround for compatibility with the “custom ui” in Homebridge plugins like this. When I first released it they had all kinds of issues.

Have you checked the browser error logs and tried another browser?

**Comment by mjamison44** [comment]
I’m using the Hoobs desktop app for mac. I’ve never had this issue. I’ve been using this plugin for years. I just switched from a raspberry pi 3 to a pi 4b. This issue started after I switched to the 4B. During the install of the plugin it gives me multiple errors.

“Warning homebridge-simplisafe3 > cli-ux@5.6.7: Package no longer supported. Contact support at https://www.npmjs.com/support for more info.”
